packages/application-shell/src/hooks/use-applications-menu/use-applications-menu.spec.tsx

```diff
@@ -222,25 +222,17 @@ describe('for production usage', () => {
       console.error = jest.fn();
       mocked(reportErrorToSentry).mockClear();
       const error = new Error('Oops');
-      renderApp(
-        <AppBarTest
-          environment={environment}
-          queryOptions={{
-            onError: reportErrorToSentry,
-          }}
-        />,
-        {
-          disableRoutePermissionCheck: true,
-          mocks: [
-            {
-              request: {
-                query: FetchApplicationsMenu,
-              },
-              error,
+      renderApp(<AppBarTest environment={environment} />, {
```

Comment:
I've just removed the `queryOptions` from this part of the test suite and now the `error` test is failing but I don't understand why.

@emmenko could you please help here? 🙏

Reply:
Do you mean this?

<img width="924" alt="image" src="https://github.com/user-attachments/assets/8eab7b15-c1c1-4c5e-ae27-7fd46f5ba727" />

We can remove that test, as it's not a concern anymore of this component / hook.

Reply:
Thanks for the clarification.
Updated here: 0307dd551870c0fed7bbfa9703f334fb1aeeb8c2